build/gulp/style-compiler.js

```diff
@@ -20,6 +20,15 @@ const autoPrefix = new LessAutoPrefix({ browsers: browsersList });
 const cssArtifactsPath = path.join(process.cwd(), 'artifacts', 'css');
 const commentsRegex = /\s*\/\*[\S\s]*?\*\//g;
 
+const DEFAULT_DEV_BUNDLE_NAMES = [
+    'common',
+    'light',
+    'material.blue.light',
+    'ios7.default'
```

Comment:
We don't need ios theme in any ci tasks.

Reply:
```
Tests run: 4325, Failures: 12, Not run: 0
--------------------------------------------------------------------------------
DevExpress.knockout/fieldset_bundled.ios7.tests.js - Horizontal align for different widgets on field-value
DevExpress.knockout/fieldset_bundled.ios7.tests.js - Equal width for same widgets on and in field-value
DevExpress.knockout/fieldset_bundled.ios7.tests.js - dxTextbox on Field
DevExpress.knockout/fieldset_bundled.ios7.tests.js - dxTextbox in Field
DevExpress.knockout/fieldset_bundled.ios7.tests.js - dxAutocomplete on Field
DevExpress.knockout/fieldset_bundled.ios7.tests.js - dxAutocomplete in Field
DevExpress.knockout/fieldset_bundled.ios7.tests.js - dxLookup on Field
DevExpress.knockout/fieldset_bundled.ios7.tests.js - dxLookup in Field
DevExpress.knockout/fieldset_bundled.ios7.tests.js - dxNumberbox on Field
DevExpress.knockout/fieldset_bundled.ios7.tests.js - dxNumberbox in Field
DevExpress.knockout/fieldset_bundled.ios7.tests.js - dxTextarea on Field
DevExpress.knockout/fieldset_bundled.ios7.tests.js - dxTextarea in Field

```